### ISSUE TITLE
Compare metric computation to TOFU

### ISSUE THREAD

**Issue by matoototo** [matoototo/llm_unlearning#6]
Ideally starts from the same plausible model output (logits) and compares results.
